Packages/ClientRuntime/Sources/Networking/Http/UnknownHttpServiceError.swift

```diff
@@ -19,9 +21,15 @@ public struct UnknownHttpServiceError: HttpServiceError, Swift.Equatable {
 }
 
 extension UnknownHttpServiceError {
-    public init(httpResponse: HttpResponse, message: String? = nil) {
+    public init(httpResponse: HttpResponse, message: String? = nil, errorCode: String? = nil) {
```

Comment:
Mind adding docs for this initializer while you're here?

Reply:
Added